dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/FATSuite.java

```diff
@@ -39,4 +45,8 @@
 
     private static final Class<?> c = FATSuite.class;
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
+                    .andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("servlet-4.0").alwaysAddFeature("servlet-5.0").fullFATOnly());
+
```

Comment:
In general we have been updating the RepeatTests to run the withoutModification in fullFatOnly and running the Jakarta EE 9 version in lite mode.

Reply:
I agree. However, a number of tests fail because they rely on servers that use mpMetrics-2.3.  Since, there's no EE9 equivalent yet, I chose to run the regular tests to run in LITE mode since there are more of them (40 EE8 vs the 13 EE9).  Once the other features are jakartafied, we can update the test run.

Reply:
If it is that few, might as well run them in lite mode also unless those 13 tests add that much to the time it takes to complete.

Reply:
I will check. It shouldn't add too much time and  it would be nice to have both run in LITE.

Reply:
Not a dramatic difference to the times. I've removed the fullFatOnly() call to allow EE9 in LITE mode. 

On lite mode added 40-50 seconds of actual test time  (excluding setup, transformations etc).
As for full mode, the new repeat code adds 2-3 minutes of test time.